src/targets/typescript.ts

```diff
@@ -78,9 +78,10 @@ export default async function build({
       );
     }
 
-    let tsc =
-      path.join(root, 'node_modules', '.bin', 'tsc') +
-      (platform() === 'win32' ? '.cmd' : '');
+    const tscPath = options?.tsc
+      ? options.tsc
```

Comment:
If a relative path is specified, we should probably resolve it to be relative to `root`

Reply:
I've added  a path resolve in my latest commit: 
```
let tsc = options?.tsc
      ? path.resolve(root, options.tsc)
      : path.join(root, 'node_module') + (platform() === 'win32' ? '.cmd' : '');
```